sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -358,7 +366,7 @@ private void LogFailedCbdeRun(Process pProcess)
             failureString.Append("  content of stderr is:\n" + pProcess.StandardError.ReadToEnd());
             failureString.Append("  content of the CBDE handler log file is :\n" + logStringBuilder.ToString());
             Log(failureString.ToString());
-            Console.Error.WriteLine($"Error when executing CBDE, more details in {cbdeProcessSpecificPath}");
+            throw new Exception($"CBDE external process reported an error.");
```

Comment:
why remove the path? it may be useful...

Reply:
or when throwing, the logs won't be dumped to the file anymore?

Reply:
because the path will be there only if the logging is enabled by an internal env variable that we would use as a support tool, right?
I can add it only if the env var is defined.

Reply:
done